src/queries/rum-pageviews-bdf.sql

```diff
@@ -0,0 +1,39 @@
+--- description: Show page views for specified domains for specified dates based on extrapolation from RUM data
+--- Authorization: none
+--- interval: 30
+--- offset: 0
+--- startdate: 2023-05-01
+--- enddate: 2023-06-01
+--- timezone: UTC
+--- url: -
+--- device: all
+--- domainkey: secret
+
+WITH rum AS (
+  SELECT
+    REGEXP_REPLACE(hostname, 'www.', '') AS hostname,
+    COUNT(DISTINCT id) AS rum_count,
+    weight,
+    FORMAT_DATE('%Y-%m-%d', time) AS date
```

Comment:
```suggestion
    FORMAT_DATE('%F', time) AS date
```


%F | DATEDATETIMETIMESTAMP | The date in the format %Y-%m-%d.
-- | -- | --





%F | DATEDATETIMETIMESTAMP | The date in the format %Y-%m-%d.
-- | -- | --